### PR TITLE
Translate 2.6.0 Released Post (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_cn/news/_posts/2018-12-25-ruby-2-6-0-released.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+## 性能提升
+
+* 由于移除了对 `$SAFE` 临时赋值的支持，提升 `Proc#call` 的速度。[[功能 #14318]](https://bugs.ruby-lang.org/issues/14318)
+
+  通过 `lc_fizzbuzz` 多次使用 `Proc#call` 的 benchmark 我们测量到了 1.4 倍性能提升 [[漏洞 #10212]]((https://bugs.ruby-lang.org/issues/10212))。
+
+* 提升了当 `block` 是代码块参数时 `block.call` 的性能。[[功能 #14330]]((https://bugs.ruby-lang.org/issues/14330))
+
+  通过与 Ruby 2.5 中引入的提升代码块传递的性能的方法结合，Ruby 2.6 进一步提升了传递代码块调用时的性能。通过 micro-benchmark 我们观察到了 2.6 倍性能提升。[[功能 #14045]](https://bugs.ruby-lang.org/issues/14045) 另外，。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2018-12-25-ruby-2-6-0-released.md]: Maybe remove "另外，。"？

**Comment by dsh0416** [comment]
@hlcfan Thanks for reviewing. I've fixed the problem mentioned.

**Comment by stomar** [comment]
Note a763274efd88580046234bece7dfe0ca51762bff.

And also please do not mix translation PR with fixes for different posts/pages.

**Comment by dsh0416** [comment]
@stomar I've fixed the trailing space, removed mixed pages and updated the download link.

**Comment by hlcfan** [comment]
Looking good, thanks!

**Pull request merged**